README.md

```diff
@@ -1,38 +1,73 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+<div align="center">
 
-## Getting Started
+<img src="https://i.imgur.com/iQgBXbj.png" alt="Logo Shoppingify Client" />
 
-First, run the development server:
+</div>
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-```
+---
+
+## :book: Overview
+
+![Imgur](https://i.imgur.com/GounMss.png)
+
+Shoppingify allows you to take your shopping list wherever you go. With Shoppingify, you can easily create personalized shopping lists for any occasion, from weekly grocery shopping to birthday gifts. You can also organize your lists by categories and add useful notes, such as product sizes and preferences.
+
+---
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## :computer: Technologies
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+- <img src="https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png" alt="Logo do typescript" width="13"> Typescript
+- <img src="https://cdn-icons-png.flaticon.com/512/875/875209.png" alt="Logo do react.js" width="13"> React
+- :zap: ChakraUI
+- <img src="https://cdn.worldvectorlogo.com/logos/next-js.svg" alt="Logo do next.js" width="13"> Next
```

Comment:
```suggestion
- <img src="https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png" alt="Logo do typescript" width="13"> Typescript

- <img src="https://cdn-icons-png.flaticon.com/512/875/875209.png" alt="Logo do react.js" width="13"> React

- :zap: ChakraUI

- <img src="https://cdn.worldvectorlogo.com/logos/next-js.svg" alt="Logo do next.js" width="13"> Next
```

Keeping an empty line in .md lists makes it easier to read for those who will edit the file. Visually doesn't actually change anything